networking_ccloud/ml2/mech_driver.py

```diff
@@ -51,12 +50,27 @@ def _plugin(self):
             self._plugin_property = directory.get_plugin()
         return self._plugin_property
 
-    def start_driver_rpc(self):
+    def start_rpc_listeners(self):
+        """Start the RPC listeners.
+
+        Most plugins start RPC listeners implicitly on initialization.  In
+        order to support multiple process RPC, the plugin needs to expose
+        control over when this is started.
+        """
+        LOG.debug("Starting cc-fabric internal driver RPC")
         self.conn = n_rpc.Connection()
         self.conn.create_consumer(cc_const.CC_DRIVER_TOPIC, [self])
 
         return self.conn.consume_in_threads()
 
+    def get_workers(self):
+        """Get any NeutronWorker instances that should have their own process
+
+        Any driver that needs to run processes separate from the API or RPC
+        workers, can return a sequence of NeutronWorker instances.
+        """
+        return [service.RpcWorker([self], worker_process_count=0)]
```

Comment:
The `worker_process_count=0` is a little confusing for someone without knowledge of the topic.

Reply:
Apparently this is the upstream way, e.g. https://github.com/sapcc/neutron/blob/ce33533c8a7e50c0b13546b5ca6836e3e14a87c3/neutron/services/l3_router/l3_router_plugin.py#L134

Reply:
Extremely confusing, but thank you for digging that out. :+1: